Source/WarpX.cpp

```diff
@@ -1116,21 +1117,6 @@ WarpX::RefRatio (int lev)
     return GetInstance().refRatio(lev);
 }
 
```

Comment:
This is not needed anymore since:
- `EvolveEM` has been renamed as `EvolveE`
- `EvolveES` is not used anymore.

Reply:
> EvolveEM has been renamed as EvolveE

do you mean
> EvolveEM has been renamed as Evolve

?